modules/apps/object/object-api/src/main/java/com/liferay/object/exception/ObjectFieldDefaultValueException.java

```diff
@@ -1,39 +0,0 @@
-/**
- * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
- *
- * This library is free software; you can redistribute it and/or modify it under
- * the terms of the GNU Lesser General Public License as published by the Free
- * Software Foundation; either version 2.1 of the License, or (at your option)
- * any later version.
- *
- * This library is distributed in the hope that it will be useful, but WITHOUT
- * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
- * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
- * details.
- */
-
-package com.liferay.object.exception;
-
-import com.liferay.portal.kernel.exception.PortalException;
-
-/**
- * @author Marco Leo
- */
-public class ObjectFieldDefaultValueException extends PortalException {
```

Comment:
Can we really delete this exception? Could you check if this has been defined in the `service.xml`?

I believe this class will be created again if you run the command `gw buildService`. Could you please check it?

Reply:
We don't need this Exception anymore. The DefaultValue was remove as parameter of ObjectField. Now, the DefaultValue is a parameter of ObjectFieldSettings.

I will remove the `ObjectFieldDefaultValue` in the `service.xml`.